src/System.Threading.AccessControl/src/System/Threading/EventWaitHandleAcl.net46.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Security.AccessControl;
+
+namespace System.Threading
+{
+    public static class EventWaitHandleAcl
+    {
+        public static EventWaitHandle Create(
+            bool initialState,
+            EventResetMode mode,
+            string name,
+            out bool createdNew,
+            EventWaitHandleSecurity eventSecurity)
+        {
+            if (eventSecurity == null)
```

Comment:
I wouldn't do any validation here that the method we call is going to do anyway. And in this case, you're allowed to pass null for security, it just assumes default. I would follow suit with the constructor for .NET- allow the null.

Reply:
Makes sense. Removing.